admiral/pkg/clusters/endpointSuspender.go

```diff
@@ -0,0 +1,5 @@
+package clusters
+
+type EndpointSuspender interface {
```

Comment:
Refactor this to EndpointUpdateSuspender?

Reply:
How about `ServiceEntryUpdateSuspender` ?